src/__tests__/initRules.test.js

```diff
@@ -510,6 +513,35 @@ describe('initRules', function() {
         expect(callOrder).toEqual(['Event1', 'Condition1', 'Action1']);
       });
 
+      it('ceases to execute remaining actions when an action throws an error', function() {
+        var rules = setupRules([
+          {
+            actions: [
+              generateAction('Action1', function(module) {
+                module.exports = jasmine.createSpy().and.callFake(function() {
+                  throw new Error('noob tried to divide by zero.');
+                });
+              }),
+              generateAction('Action2', function(module) {
+                module.exports = jasmine.createSpy();
+              })
+            ]
+          }
+        ]);
+
+        runInitRules(rules);
+
+        var action1Export = moduleProvider.getModuleExports(
+          moduleHelper.getPath('Action1')
+        );
+        var action2Export = moduleProvider.getModuleExports(
+          moduleHelper.getPath('Action1')
+        );
+
+        expect(action1Export.calls.count()).toBe(1);
+        expect(action2Export.calls.count()).toBe(1);
+      });
+
```

Comment:
I think we need the same test for when the queue is not being used.

Reply:
Oh nevermind. I misunderstood.

Reply:
I think I means...we need the same test for when the queue IS being used.

Reply:
Added a test for the queue set. FIxed also this test which was wrong.